pom.xml

```diff
@@ -20,7 +20,7 @@
     <groupId>ons.sdc.int.common</groupId>
     <artifactId>common</artifactId>
     <!-- change this to the version of ALL common libs to be used by this build -->
-    <version>1.0.44</version>
+    <version>1.0.48</version>
```

Comment:
we don't need the PGP stuff anymore so could remove the bouncycastle dependency

Reply:
Removed

Reply:
Removed